### ISSUE TITLE
R installation?

### ISSUE THREAD

**Issue by jovo** [flashxio/knor#51]
@disa-mhembere tried to install R version, couldn't find instructions anywhere.

**Comment by disa-mhembere** [comment]
I think `install.packages("knor")` may work now. I received notification it
was on cran but haven't tried yet. I'll document once I do though.

**Comment by jovo** [comment]
```
Warning messages:
1: In doTryCatch(return(expr), name, parentenv, handler) :
  unable to load shared object '/Library/Frameworks/R.framework/Resources/modules//R_X11.so':
  dlopen(/Library/Frameworks/R.framework/Resources/modules//R_X11.so, 6): Library not loaded: /opt/X11/lib/libSM.6.dylib
  Referenced from: /Library/Frameworks/R.framework/Resources/modules//R_X11.so
  Reason: image not found
2: package ‘knor’ is not available (for R version 3.4.1) 
> 
```

**Comment by disa-mhembere** [comment]
`install.packages("knor")` now is verified to work.
